proto/decentraland/kernel/comms/rfc4/comms.proto

```diff
@@ -14,9 +14,45 @@ message Packet {
     Chat chat = 5;
     Scene scene = 6;
     Voice voice = 7;
+    SdkComponent sdk_component = 8;
+    CustomSignal signal = 9;
   }
 }
 
+// custom signal is used to encode any kind of custom signal that is not part of
+// the standard protocol. The name field is used to identify the signal type, and
+// the data field contains the serialized signal data.
+// A concrete example of this is the ping signal, which is used to measure the
+// latency of the connection. The name field would be "ping", and the data field
+// would contain the serialized Ping message.
+// This message is not part of the standard protocol, and is only used to encode
+// custom implementation-dependant signals.
+//
+// It is recommended that signal names are namespaced to avoid collisions. For
+// example, a signal to request a profile could be named "decentraland.profile-request.v1"
```

Comment:
This recommendation is super reasonable. Why not enforce it by having a `namespace` field?